install_sct.bat

```diff
@@ -0,0 +1,45 @@
+:: Usage: install_sct.bat <version>
+:: e.g.
+::        install_sct.bat 5.5
+
+:: Fetch version number (should be git branch, git tag, etc.)
+set git_ref=%1
+
+:: Save present working directory to return to later
+FOR /F "tokens=*" %%G IN ('cd') do (SET PWD=%%G)
+
+:: Go to user's home directory
+cd %HOMEPATH%
+
+:: Download SCT and check out the branch requested by the user
+git clone -b %git_ref% --single-branch --depth 1 https://github.com/spinalcordtoolbox/spinalcordtoolbox.git || goto :error

```

Comment:
My thought process with cloning is that we would attach `install_sct.bat` to a release, and the user would download _just_ the lightweight script.

Ideally, we would do this for `install_sct`, too. That way we would address https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3545 for all platforms.